src/sst/core/interprocess/mmapparent.cc

```diff
@@ -0,0 +1,15 @@
+// Copyright 2009-2019 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2019, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#include "sst_config.h"
```

Comment:
Why do we need an implementation file that just includes the header?

Reply:
The only reason is to check for syntax errors and such during the core compile, otherwise those aren't caught until sst-elements tries to use it. I don't see a use outside of development, so we could remove them too.

Reply:
Okay. No, I like the strategy actually for debugging in core. We could also do an explicit template instantiation in this file so we also "debug" a template instantiation.

Reply:
If this file is strictly for catching errors in the header file at compile time, I would add a comment to that effect right after the copyright info.  I would also be nice, though not strictly necessary, to have the file compile using a makefile target that won't link the resulting object file with the executable.

Reply:
There doesn't look to be an easy way to keep the object out of the executable. I will add a comment in these files that they are only for testing and do an explicit instantiation as well.